apps/tsilhqotin-language-hub/src/Components/Pages/Apps/AppsDetail.tsx

```diff
@@ -0,0 +1,101 @@
+import { Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material';
+
+export function AppsDetail() {
+    const appData = [
+        {
+            name: 'Digital Phrasebook v1.0',
+            image: 'https://www.tsilhqotin.ca/wp-content/uploads/2022/08/dpb_logo8.png',
+            meta: 'released 2022',
+            description:
+                "Tŝilhqot'in Dictionary Project contains over 10,000 words and paradigms. *Logo is a prototype",
+            links: {
+                google: 'https://play.google.com/tsilhqotin',
+                web: 'https://weblink.com/tsilhqotin',
+            },
+        },
+        {
+            name: 'Digital Verb Book',
+            image: 'https://api.tsilhqotinlanguage.ca/uploads/app_icon_dvb_483177a8a0.png',
+            meta: 'released 2018',
+            description:
+                'Version 1.0 is currently published and contains "Third Person Singular Paradigms" completed Bella Alphonse. These are really helpful for semi-speakers who need practice with verb stem alternations. To date over 100,000 phrases have been recorded, with about 50,000 named and organized. Version 2.0 of this app will allow us to present this material to learners. This is the main focus of the tech team for 2021-2022. ',
+            links: {
+                google: 'https://play.google.com/tsilhqotin',
```

Comment:
I probably designed this data structure, but I'd prefer to have it slightly differently:
```ts
type AppLink = {
platform: 'google' | 'web' | 'ios';
link: string; // URL
}

type AppInfo = {
...,
links: AppLink[]
}
```

Reply:
We did this.